packages/pigeon/lib/java_generator.dart

```diff
@@ -113,6 +113,19 @@ void _writeCodec(Indent indent, Api api) {
   });
 }
 
+String _boxedType(String type) {
```

Comment:
Could you add a declaration comment here? It's non-obvious from the name that "boxing" a type that's not on the list is actually a no-op.

Reply:
Added comments.

It was a mistake adding this method.  I started removing it in later PR's.  The way things are written there needs to to be a function like this that can return null and another that does a passthrough depending on the usage.  This should be joined with `_javaTypeForDartType` since it is duplication.

Reply:
I removed this function completely and moved to _javaTypeForDartType in this PR: https://github.com/flutter/packages/pull/418